src/components/Search.vue

```diff
@@ -16,11 +16,48 @@
     <a @click="clearInput">
       <Icon v-if="modelValue" name="close" size="12" class="mb-1" />
     </a>
+    <UiDropdown
```

Comment:
This should no be embed into `<Search/>` component, you can create a component `<SearchWithFilters/>` that include the Search component.